src/contexts/ErrorContext/ErrorContext.tsx

```diff
@@ -10,18 +10,26 @@ import {
 } from "react";
 import { useModalContext } from "../ModalContext";
 
-type State = { handleError: (error: any, displayMessage?: string) => void };
+type State = {
+  handleError: (
+    error: any,
+    displayMessage?: string,
+    options?: { log: boolean }
```

Comment:
@ap-justin using this `options` param can allow us to reuse the error prompt logic without logging the error (referencing the problem you raised here https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2753#discussion_r1509096323